source/sfp/1_water/sfp_114_nl_waterstaalname_plas/06_benodigdheden.Rmd

```diff
@@ -0,0 +1,113 @@
+# Benodigdheden
+
+Tabel 1 geeft een overzicht van de benodigde apparatuur en materiaal voor het nemen van een waterstaal.
+Onder 7.1, 7.2 en 7.3 worden een aantal specifieke benodigdheden van verdere uitleg voorzien.
+
++:------------------------------------------------------------+:--------------------------------------------------------------------------+
+| -   staalnameformulieren & protocol of digitaal alternatief | -   staalname-emmer                                                       |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   klembord, potlood en slijper                            | -   maatbeker                                                             |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   laarzen, lieslaarzen, waadpak (plaatsafhankelijk)       | -   spuiten en filters                                                    |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   latex handschoenen (poederloos)                         | -   koelbox met koelelementen                                             |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   plooimeter                                              | -   reserverecipiënten en hersluitbare zakjes                             |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   fototoestel                                             | -   documentatie van het staalnamepunt om dit exact te kunnen lokaliseren |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   recipiënten                                             | -   hamer om een gat in het ijs te maken                                  |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+
+Tabel 1: Checklist benodigd veldmateriaal.
+
+## Apparatuur
+
+Niet van toepassing
+
+## Materiaal
+
+**Staalname-emmer**
+
+Witte emmer (polyethyleen), inhoud 10 l, met maatmarkering en giettuit (figuur 1).
```

Comment:
best overal gebruik maken van kruisverwijzigingen; wellicht iets in de aard van \@ref(label) --> zie https://bookdown.org/yihui/bookdown/figures.html#fn6 of voorbeeld uit een ander protocol

Reply:
Is nu in orde voor de drie figuren en twee tabellen (moest eerst nog een "caption" toewijzen via (#fig:caption) of (#tab:caption), nu werkt het normaal gezien dus wel)